docs/source/science/NISAR/OPERA Surface Displacement.ipynb

```diff
@@ -0,0 +1,4440 @@
+{
```

Comment:
<p>Should use same plotting parameters as the local file variant in order to show they are identical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>